concourse/pipelines/deploy.yml

```diff
@@ -0,0 +1,165 @@
+---
+definitions:
+
+resource_types:
+  - name: slack-notification
+    type: docker-image
+    source:
+      repository: cfcommunity/slack-notification-resource
+      tag: latest
+
+resources:
+  - &git-repo
+    icon: github
+    name: govuk-infrastructure
+    source:
+      branch: main
+      uri: https://github.com/alphagov/govuk-infrastructure
+    type: git
+
+  - <<: *git-repo
+    name: frontend
+    source:
+      branch: master
+      uri: https://github.com/alphagov/frontend
+      tag_filter: release_*
+
+  - <<: *git-repo
+    name: publisher
+    source:
+      branch: master
+      uri: https://github.com/alphagov/publisher
+      tag_filter: release_*
+
+  - <<: *git-repo
+    name: publishing-api
+    source:
+      branch: master
+      uri: https://github.com/alphagov/publishing-api
+      tag_filter: release_*
+
+  - <<: *git-repo
+    name: content-store
+    source:
+      branch: master
+      uri: https://github.com/alphagov/content-store
+      tag_filter: release_*
+
+  - name: deploy-slack-channel
+    type: slack-notification
+    source:
+      url: https://hooks.slack.com/services/((slack_webhook))
+
+groups:
+  - name: all
+    jobs:
+      - update-pipeline
+      - deploy-content-store
+      - deploy-frontend
+      - smoke-test-frontend
+      - deploy-publisher
+      - deploy-publishing-api
+
+  - name: admin
+    jobs:
+      - update-pipeline
+
+  - name: content-store
+    jobs:
+      - deploy-content-store
+
+  - name: frontend
+    jobs:
+      - deploy-frontend
+      - smoke-test-frontend
+
+  - name: publisher
+    jobs:
+      - deploy-publisher
+
+  - name: publishing-api
+    jobs:
+      - deploy-publishing-api
+
+jobs:
+  - name: update-pipeline
+    plan:
+    - get: govuk-infrastructure
+      trigger: true
+    - file: govuk-infrastructure/concourse/pipelines/cd-frontend.yml
```

Comment:
I couldn't find this file - am I missing something? :)

Reply:
Thanks, fixed in superseding PR :)